### PR TITLE
Update Strings for Simplified Chinese

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -1,123 +1,196 @@
...
+	<string name="credits_content5">感谢 <a href="https://github.com/psyanite">Yan Tsui</a>
+		测试并设计了&appname; 的图标。
 	</string>
-
-	<string name="credits_content6"><a href="https://github.com/wardbradt">Ward Bradt</a>
-		把&appname;从英文翻译成中文。&appname;的此版本翻译由<a href="https://github.com/Liuxi-G">Liuxi-G</a>
-		完成。
+	<string name="credits_content6"><a href="https://github.com/wardbradt">Ward Bradt</a>负责初版
+	简中翻译，<a href="https://github.com/Liuxi-G">Liuxi-G</a>进行了后续完善。
 	</string>
```
Comment (ScreamingHawk) [inline app/src/main/res/values-zh/strings.xml:81]: This section is to give credits to the translators. Please add yourself to this list if you want to be credited in the app :)

**Comment by ScreamingHawk** [comment]
Thanks so much for the pull request!

Added a comment around the credits. If you don't want to be credited let me know and I can merge this through

**Comment by sr093906** [comment]
Thanks. I forgot to mention that in the string. Please merge the pull request.

**Comment by ScreamingHawk** [comment]
Done. Thanks again for your contribution! Translations can be an easy thing for people who speak the language but it's impossible for me to do alone. I really appreciate it.

**Pull request merged**